views/admin/assignments.html

```diff
@@ -33,15 +33,31 @@
 <script src="{{=URL('static', 'js/bootstrap-table/bootstrap-table-reorder-rows.js')}}">
 </script>
 
-<div class="col-md-6">
-  <h4 style="text-align: center;">Assignment</h4>
-  <select class="form-control" id="assignlist" onchange="assignmentInfo();">
-    {{ for assignmentid in assignments: }}
-    <option value="{{=assignmentid}}">{{=assignments[assignmentid]}}</option>
-    {{ pass }}
-  </select>
-  <button class="btn" data-target="#addAssignment" data-toggle="modal" style="margin-top: 12px;">+</button>
-  <button class="btn" onclick="remove_assignment();" style="margin-top: 12px;">-</button>
+<div class="row">
```

Comment:
I'll study these a bit more, but as I look at the interface, it seems that the ability to write your own questions for an assignment is completely gone.  Why?